src/code/code_800E4FE0.c

```diff
@@ -38,6 +38,9 @@ AudioTask* func_800E4FE0(void) {
 extern u64 rspAspMainDataStart[];
 extern u64 rspAspMainDataEnd[];
 
+/**
+ * Updates audio functions run on the audio thread
```

Comment:
is this to be understood as "this function updates audio functions that run on the audio thread" (how I understand this comment 😅 )
or as "this is an update audio function that runs on the audio thread" (what I think this function is?)

same on `func_800F3054` I didn't see before 🙃

Reply:
I'm not sure I'm following what the distinction is, update functions are fairly well-defined all throughout the repo. This is the highest level audio-update function on the audio thread, which would just be called `Audio_Update()` like most systems. It's just that audio is divided between two threads, so there would be both an `Audio_GraphThreadUpdate()` and a `Audio_AudioThreadUpdate()` as a verbose name example, but finding a good name is outside the scope of this PR

Reply:
I'm just discussing grammar, the current wording I understand as "updating the functions" instead of "a function that updates" 😅